charts/tidb-operator/values.yaml

```diff
@@ -151,7 +151,7 @@ apiserver:
 #   kubectl apply -f manifests/advanced-statefulset-crd.v1.yaml      # k8s version >= 1.16.0
 advancedStatefulset:
   create: false
-  image: pingcap/advanced-statefulset:v0.3.3
+  image: pingcap/advanced-statefulset:v0.4.0
```

Comment:
This new image contains tzdata, so that we can use `TZ` environment to configure the local time zone.